bodhi/server/consumers/composer.py

```diff
@@ -232,29 +232,29 @@ def work(self, msg):
         """Begin the push process.
 
         Here we organize & prioritize the updates, and fire off separate
-        threads for each repo tag being mashed.
+        threads for each repo tag being composed.
 
         If there are any security updates in the push, then those repositories
         will be executed before all others.
         """
         body = msg['body']['msg']
         resume = body.get('resume', False)
         agent = body.get('agent')
-        notifications.publish(topic="mashtask.start", msg=dict(agent=agent), force=True)
+        notifications.publish(topic="composetask.start", msg=dict(agent=agent), force=True)
```

Comment:
We could probably drop ```task``` from this too:

```suggestion
        notifications.publish(topic="compose.start", msg=dict(agent=agent), force=True)
```